pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -259,6 +265,96 @@ func (vm *Windows) ConfigureCNI(configFile string) error {
 	return nil
 }
 
+// service represents a Windows service
+type service struct {
+	// binPath includes a path to the binary and all arguments to run the binary with
+	binPath string
+	// name is the name of the service
+	name string
+	// vm is the VM the service will be started on
+	vm *Windows
+}
+
+// newService returns a pointer to a new service
+func newService(vm *Windows, name, binPath string) *service {
+	return &service{
+		vm:      vm,
+		name:    name,
+		binPath: binPath,
+	}
+}
+
+// start creates and starts the service on the VM
+func (s *service) start() error {
+	_, stderr, err := s.vm.Run("sc.exe create "+s.name+" binPath=\""+s.binPath+"\" start=auto", false)
```

Comment:
Is there no info returned to `stdout` that we might want to see in the case of an error?